client/src/reducers/auth.js

```diff
@@ -124,22 +28,19 @@ export const authSlice = createSlice({
       state.token = null;
       state.user = null;
     },
-    loginFail: (state, action) => {
```

Comment:
Since we are not passing any payload when login fails, then the second param is not used or necessary